openassessment/locale/ar/LC_MESSAGES/django.po

```diff
@@ -308,15 +310,6 @@ msgid ""
 "assessed sample responses that the instructor provides."
 msgstr "تقيّيم الخوارزمية ردود المتعلّمين وفقًا لعملية مقارنة بين الردود ونماذج ردود مقيّمة مسبقًا يوفّرها موجّه المساق."
 
-#: openassessment/templates/openassessmentblock/edit/oa_edit_example_based_assessment.html:16
-#: openassessment/templates/openassessmentblock/edit/oa_edit_student_training.html:18
-msgid ""
-"Enter one or more sample responses that you've created, and then specify the"
-" options that you would choose for each criterion in your rubric. Note that "
-"you must add your rubric to the Rubric tab before you can complete this "
-"step."
-msgstr "أدخِل نموذج ردود واحد أو أكثر ممّا أنشأته، ثمّ حدّد الخيارات التي قد تنتقيها لكل معيار في سلّم تقييمك. ويُرجى ملاحظة أنّه عليك أن تضيف سلّم تقييمك إلى تبويبة سلّم التقييم قبل أن تتمكّن من استكمال هذه الخطوة."
-
```

Comment:
This stanza was not removed; it has just been moved [down to here by `master`](https://github.com/edx-olive-oc/edx-ora2/pull/4/files#diff-5c9081c9e1c719e582598a18d7a6cef5R493).